lens-cube/src/main/java/org/apache/lens/cube/parse/StorageCandidate.java

```diff
@@ -518,13 +575,44 @@ private boolean evaluateMeasuresCompleteness(TimeRange timeRange) throws LensExc
   }
 
   @Override
-  public Set<FactPartition> getParticipatingPartitions() {
-    return null;
+  public boolean isExpressionEvaluable(ExpressionResolver.ExpressionContext expr) {
+    return expr.isEvaluable(this);
   }
 
   @Override
-  public boolean isExpressionEvaluable(ExpressionResolver.ExpressionContext expr) {
-    return expr.isEvaluable(this);
+  public void updateAnswerableSelectColumns(CubeQueryContext cubeql) throws LensException {
```

Comment:
Can we leave a method level comment explaining what we are trying to achieve in this method and at what stage it will be called.